CONTRIBUTING.md

```diff
@@ -256,6 +256,16 @@ run only the unit tests use:
 
     TEST_TARGET=tests/unit make check-in-container
 
+#### **db tests**
```

Comment:
nit: Can we call the section "Database tests" please? So that it's obvious after reading the header